aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -97,6 +106,7 @@ class QLinearPackWeightInt8 final : public c10::OperatorKernel {
             /*ld=*/K,
             /*pmat=*/nullptr, // PackBMatrix manages ownership of pmat
             /*groups=*/1),
+        bias_contig,
```

Comment:
How does this work when bias is None?

Reply:
It is stored as an optional tensor - similar to the current linear op that has bias as an input argument. So if it is none then that gets taken care of.